docs/running/utils.rst

```diff
@@ -223,6 +223,21 @@ Otherwise, the ``toil status`` command will return something like the following:
 
 The ``toil status`` command supports several useful flags, including ``--perJob`` to get per-job status information, ``--logs`` to print stored worker logs, and ``--failed`` to list all failed jobs in the workflow. For more information, run ``toil status --help``.
 
+One use case of ``toil status`` is with the ``--printStatus`` argument. Running ``toil status --printStatus file:my-jobstore`` at any point of the workflow's lifecycle can tell you the progress of the workflow.
+Note: This command will output all current running jobs but not any finished or failed jobs.
```

Comment:
I think it would be useful if the `--printStatus` command also outputted the number of finished jobs, such as with failed jobs in line 233.